.github/workflows/tests.yml

```diff
@@ -146,12 +126,13 @@ jobs:
 
       - name: Install dependencies
         run: |
-          pip install -r .${{matrix.requirements-level}}-${{ matrix.python-version }}-requirements.txt
-          pip install .[${EXTRAS}]
-          pip freeze
-          docker --version
-          docker-compose --version
+          if [ ${E2E} == 'yes' ] && [ ! -f $HOME/webdrivers/chromedriver ]; then wget https://chromedriver.storage.googleapis.com/2.31/chromedriver_linux64.zip -P $HOME/webdrivers; unzip -d $HOME/webdrivers $HOME/webdrivers/chromedriver_linux64.zip; fi
+          nvm install 14.0.0; nvm use 14.0.0
```

Comment:
Maybe you can use something like [`setup-node`](https://github.com/marketplace/actions/setup-node-js-environment) action instead (untested)?